### ISSUE TITLE
It is possible to implement alpha beta pruning? 

### ISSUE THREAD

**Issue by pedrosarkis** [adeelibr/tic-tac-toe-ai-vanilla-js#1]

**Comment by adeelibr** [comment]
Yes for sure it is, here are some alpha beta pruning code samples I found

- https://codepen.io/lycjee/pen/BRJJGo - Code sample
- https://medium.com/@alialaa/tic-tac-toe-with-javascript-es2015-ai-player-with-minimax-algorithm-59f069f46efa - blog code tutorial


Would you like to add a PR to add an example with alpha beta pruning to this codebase? :) @pedrosarkis

**Comment by pedrosarkis** [comment]
@adeelibr Yes, of course I'll try, I'm new on that type of algorithm, lot to learn.
